libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -170,6 +170,11 @@ void AP_InertialSensor_Backend::_publish_gyro(uint8_t instance, const Vector3f &
     _imu._gyro_healthy[instance] = true;
 
     // publish delta angle
+    if ((_imu._delta_angle_acc_dt[instance] > 0) && (_imu._delta_angle_dt[instance] > 0)) {
```

Comment:
Strictly this is using invalid data on the first calculation before `_delta_angle_valid` becomes true, to fix we would have to add a bool to know if the last delta angle is valid.